modules/mongodb/manifests/s3backup/cron.pp

```diff
@@ -5,13 +5,24 @@
 # [*user*]
 #   The user to run the cronjob as.
 #
-class mongodb::s3backup::cron {
+class mongodb::s3backup::cron(
+  $user = 'govuk-backup'
+) {
+
+  include backup::client
+  require mongodb::s3backup::package
+  require mongodb::s3backup::backup
 
   cron { 'mongodb-s3backup':
     command => '/usr/local/bin/mongodb-backup-s3',
-    user    => $mongodb::s3backup::backup::user,
+    user    => $user,
     minute  => '*/15',
-    require => [Class['mongodb::s3backup::package'],Class['mongodb::s3backup::backup']],
+  }
+
+  cron { 'mongodb-s3-night-backup':
+    command => '/usr/local/bin/mongodb-backup-s3 daily',
+    user    => $user,
+    hour    => '0',
```

Comment:
`minute => 0` still needs to be added. I'm also wondering whether we need to set some sort of lock as this will kick off two mongodump processes at the same time - will that work or will one error out? 

If you use `setlock -N /var/run/foo` for the daily backup that will wait until a lock is available, and `setlock -n` on the regular backup will just exit if it can't get the lock.